content/articles/build-an-ip-address-tracker/index.md

```diff
@@ -0,0 +1,773 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /ip-tracker/
+title: How to Build an IP Address Tracker using Mapbox API and Vue
+description: In this article, we are going to look at building an IP Address Tracker using Mapbox API and Vue
+author: abiola-farounbi
+date: 2022-01-05T00:00:00-11:10
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/ip-tracker/hero.jpg
+    alt: How to Build an IP Address Tracker with Mapbox API and Vue hero image
+---
+
+An IP(Internet Protocol) address is a set of numbers that represents the location website or device. Each number in the set ranges from 0 to 255, that is from 0.0.0.0 to 255.255.255.255.
+
+They are of two types- IPv4 and IPv6.
+IPv4 is old and limited to 32-bit addresses while IPv6 is the new standard and allows for 128-bit addresses. The creation of IPv6 is because all possible IPv4 addresses are close to being exhausted as many new devices are being assigned to them, so IPv6 is the new standard. Find more about IP addresses [here](https://www.elegantthemes.com/blog/wordpress/what-is-an-ip-address).
+
+The IP Address of a user can be used for various dynamic purposes one of which is **IP Geolocation**. This is the identification of a geographic location of a device by using an IP address. This involves tracking the device based on the IP address assigned to it. The IP address contains information pointing to where the device is located such as the latitude, longitude, country, region(city), timezone, etc.

```

Comment:
add a comma after purposes. This is the indentification of a grographic location = IP Geolocation is.

Flow/readability edits.

Missing comma after region(city)